experimental/addons/redis-exporter/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+name: redis-exporter
+version: 0.0.1
+appVersion: v1.45.0
+description: Prometheus Redis metrics exporter.
+icon: https://redis.com/wp-content/themes/wpx/assets/images/logo-redis.svg
+url: https://github.com/oliver006/redis_exporter
+
+tags:
+  - Observability
+
+deployTo:
+  controlPlane: true
+  runtimeCluster: false
```

Comment:
Does this addon only need to be installed in ControlPlane?

Reply:
Yes, i think so, exporter running in the control plane can export the metrics. 

If you have different idea about it, please give me some advice. Because i 

By the way, mysql exporter addon is only needed  to deploy to control plane. So, i think redis exporter maybe the same.

Reply:
Understood. I am just concerned that this may be due to building a cross-cluster connection, which could potentially consume more bandwidth. However, I believe that we can optimize this in the future.

Reply:
ok